src/component/felles/menu/Menu.tsx

```diff
@@ -1,51 +1,40 @@
 import styles from './Menu.module.less';
-import { Normaltekst } from 'nav-frontend-typografi';
-import { useValgtVirksomhetStore } from '../../../store/valgt-virksomhet-store';
+import { Normaltekst, Systemtittel } from 'nav-frontend-typografi';
 import React from 'react';
+import cls from 'classnames';
 import { useDataStore } from '../../../store/data-store';
-import { useHistory } from 'react-router-dom';
+import { AlertStripeInfo } from 'nav-frontend-alertstriper';
+import navLogo from './nav-logo.svg';
+import { VirksomhetVelger } from './virksomhet-velger/VirksomhetVelger';
 
 export const Menu = () => {
-	const history = useHistory();
 	const { innloggetAnsatt } = useDataStore();
-	const { valgtVirksomhet, setValgtVirksomhet } = useValgtVirksomhetStore();
-
-	const tilgjengeligeVirksomheter = innloggetAnsatt.virksomheter;
-
-	const onValgtVirksomhetChanged = (e: React.ChangeEvent<HTMLSelectElement>) => {
-		const id = e.target.value;
-		const nyValgtVirksomhet = tilgjengeligeVirksomheter.find(virksomhet => virksomhet.id === id);
-
-		if (nyValgtVirksomhet) {
-			setValgtVirksomhet(nyValgtVirksomhet);
-		}
-
-		// Når vi bytter virksomhet så redirect til tiltaksinstans-oversikt hvis vi ikke allerede er der
-		if (history.location.pathname !== '/') {
-			history.push('/');
-		}
-	};
 
 	return (
 		<header className={styles.menu}>
+			<div className={styles.pilotInfoSection}>
+				<AlertStripeInfo>
+					Dette er en ny tjeneste som ikke er ferdig.
+					Ta kontakt med oss her hvis du har noen tilbakemeldinger.
+				</AlertStripeInfo>
+			</div>
+
+			<div className={styles.logoAndTitleSection}>
+				<img src={navLogo} className="blokk-xs" alt="NAV logo"/>
+				<Systemtittel>Oversikt for tiltaksarrangører</Systemtittel>
+			</div>
+
 			<div>
-				<Normaltekst className="blokk-xxs">Karoline Koordinatorsen</Normaltekst>
+				<Normaltekst className="blokk-xxs">{`${innloggetAnsatt.fornavn} ${innloggetAnsatt.etternavn}`}</Normaltekst>
+
+				<VirksomhetVelger className="blokk-xxs" />
```

Comment:
Kan vi legge til en ny blokktype i globals.module.less og bruke den her isteden for å skrive ut strengen?

Reply:
Hmmm. Føler det er litt kjipt å redefinere stylingen i globals.module.less, men det blir kanskje litt mer konsekvent å kun bruke module styling istedenfor å blande med vanlige klasser

Reply:
Ja, og syns det gir bedre kontroll over eksterne ting. Det er lettere å finne usages og potensielt gå bort ifra navs mixins om man vil det. Ser forsåvidt ikke helt poenget med at designbiblioteket skal definere hvor mye luft man skal ha, når det bare er snakk om en margin som jeg gjerne heller at man har kontroll over i egen app.

Reply:
`Ser forsåvidt ikke helt poenget med at designbiblioteket skal definere hvor mye luft man skal ha, når det bare er snakk om en margin som jeg gjerne heller at man har kontroll over i egen app.`

Dette er faktisk ganske viktig. Hvis man er konsekvent på marginer så ser designet veldig mye mer helhetlig ut.

Reply:
Men jeg kan flytte det inn i global.module.less og sjekke om det er noe blokk-styling andre steder og fikse det der også 👍